src/test/kotlin/com/atlassian/performance/tools/jiraactions/api/scenario/JiraRteEditPageTest.kt

```diff
@@ -0,0 +1,174 @@
+package com.atlassian.performance.tools.jiraactions.api.scenario
+
+import com.atlassian.performance.tools.dockerinfrastructure.api.browser.DockerisedChrome
+import com.atlassian.performance.tools.dockerinfrastructure.api.jira.JiraCoreFormula
+import com.atlassian.performance.tools.jiraactions.api.ActionMetric
+import com.atlassian.performance.tools.jiraactions.api.ActionResult
+import com.atlassian.performance.tools.jiraactions.api.SeededRandom
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import com.atlassian.performance.tools.jiraactions.api.action.*
+import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import com.atlassian.performance.tools.jiraactions.api.measure.output.CollectionActionMetricOutput
+import com.atlassian.performance.tools.jiraactions.api.memories.IssueKeyMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.User
+import com.atlassian.performance.tools.jiraactions.api.memories.UserMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.adaptive.AdaptiveIssueKeyMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.adaptive.AdaptiveIssueMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.adaptive.AdaptiveJqlMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.adaptive.AdaptiveProjectMemory
+import com.atlassian.performance.tools.jiraactions.api.w3c.DisabledW3cPerformanceTimeline
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import java.time.Clock
+import java.util.*
+
+/**
+ * Test Editing issue without SetupAction (i.e Rich Text editor enabled)
+ * Covers:
+ *      EditIssue
+ *      AddComment
+ */
+class JiraRteEditPageTest {
+
+    @Test
```

Comment:
WRT the [failed build](https://travis-ci.com/atlassian/jira-actions/builds/101537582), it didn't fit in the current 14 minute timeout. I'm running the tests locally to see how long does it take and therefore what should the timeout be.

Reply:
I will increase it by 4 mins then :)

Reply:
You can do it within this PR itself. Note the link in my comment.

Reply:
sure